Miscellaneous/Sign in with apple.md

```diff
@@ -0,0 +1,210 @@
+
+You don't have to do anything about this, system alert will automatically be shown, guiding users to the settings where they can sign in to their Apple ID.
+
+![No Apple ID][image-3]
+
+### 2nd case - user is logged into Apple ID
+
+If the user is already signed in but using sign in with Apple for the first time, they will see the Data and privacy information screen.
+
+![Data and privacy information][image-4]
+
+If they cancel here, you will get the `ASAuthorizationErrorCanceled` error case, while tapping the **Continue** button would bring them to the sign in screen. We will also get this same error case if the user cancels the flow in any other subsequent case.
+
+> Note - Every sign in request done in a simulator is treated as the first time, so you will see this dialog every time if you test in a simulator.
+
+On the sign in screen, users will be presented with a form, prefilled with the full name and email that they used to register their Apple account.
+
+Scope of the information that is shown here will depend on the scope you required when you were creating the request: 
+
+```swift
+let appleIDProvider = ASAuthorizationAppleIDProvider()
+let request = appleIDProvider.createRequest()
+request.requestedScopes = [.fullName, .email]
+```
+
+![Information scope][image-5]
+
+After the users choose their preference regarding sharing their real name and e-mail adress and press **Continue**, there are no extra steps - Apple will do their magic, authenticate the user for you, and then notify you via the `authorizationController(controller: ASAuthorizationController, didCompleteWithAuthorization authorization: ASAuthorization)` delegate method where you can get user information from the `ASAuthorization` object.
+
+You can retrieve **fullName**, **email**, **user**, and **identityToken** from the object.
+
+```swift
+func authorizationController(controller: ASAuthorizationController, didCompleteWithAuthorization authorization: ASAuthorization) {
+    if let appleIDCredential = authorization.credential as? ASAuthorizationAppleIDCredential {        
+        guard let appleIDToken = appleIDCredential.identityToken else {
+            print("Unable to fetch identity token")
+            return
+        }
+        
+        // JWT token you will probably need to send to the backend.
+        guard let idTokenString = String(data: appleIDToken, encoding: .utf8) else {
+            print("Unable to serialize token string from data: \(appleIDToken.debugDescription)")
+            return
+        }
+        
+        let userIdentifier = appleIDCredential.user
+        let fullName = appleIDCredential.fullName
+        let email = appleIDCredential.email
+        
+        print(idTokenString) 
+        print(userIdentifier)
+        print(fullName)
+        print(email)  
+    } 
+}
+```
+
+And just like that, you've just implemented your first Sign in with Apple!
+
+## Implementation notes
```

Comment:
1. Is it still relevant that you'll get user information (email, name, etc) only the first time, every other time you'll only get jwt? Please add that warning about this one.

2. I think that from iOS 14, Apple introduced something like "migration". Until now, if you already had an account (e.g. created with Google), you could only create a new one with Apple. From now, if you are already signed, you can migrate that acc from Google to Apple. Is this something relevant for us or is this something related to web implementation?